### PR TITLE
Add region

### PR DESCRIPTION
**Pull request by quarterdome**
Segment new Redshift cluster is in us-east-2 region, so we now need to specify region when we COPY from our buckets. 

@jrichard @kristjan

### CODE REVIEW

File: lib/redshifter/util/create_or_replace_table.rb [hunk 1]
```diff
@@ -49,6 +49,7 @@ def drop_and_create_table_sql
         GZIP
         TIMEFORMAT 'YYYY-MM-DD HH:MI:SS'
         NULL AS '#{ExtractAndTransformUpdates::NULL_CHARACTER}'
+        REGION 'us-east-1'
```
Comment (jrichard) [inline lib/redshifter/util/create_or_replace_table.rb]: Your PR said the new cluster is in us-east-2, not us-east-1.

**Comment by kristjan** [comment]
LGTM. This is because our S3 bucket is in `us-east-1`?

File: lib/redshifter/util/update_table.rb [hunk 2]
```diff
@@ -55,6 +55,7 @@ def create_and_load_temp_table_sql
         TIMEFORMAT 'YYYY-MM-DD HH:MI:SS'
         NULL AS '#{ExtractAndTransformUpdates::NULL_CHARACTER}'
         MANIFEST
+        REGION 'us-east-1'
```
Comment (jrichard) [inline lib/redshifter/util/update_table.rb]: Your PR said the new cluster is in us-east-2, not us-east-1.

**Comment by jrichard** [comment]
Same question as Kristjan: is this region for S3 and the cluster region is used as default, and now they differ?

I'm curious because the redshift web monitor may need to specify the region if it's not related to S3.

**Comment by quarterdome** [comment]
Yes, the cluster is in `us-east-2`, but the bucket is in `us-east-1`. If you use COPY command without specifying `us-east-1`, then you will get the following error: `S3ServiceException:The bucket you are attempting to access must be addressed using the specified endpoint`

**Comment by quarterdome** [comment]
Can I get an lg2m, if that answers your question?

**Comment by jrichard** [comment]
lgtm

**Pull request merged**